synapse/lib/modelrev.py

```diff
@@ -1510,6 +1504,9 @@ async def revModel_0_2_31(self):
 
                 refs = node['refs'].get(layer.iden, [])
 
+                assert isinstance(refs, list)
```

Comment:
The asserts are not present in production use; so these checks won't be present in at all.

Reply:
i know. it's only for testing purposes